LogglyService.xcodeproj/project.pbxproj

```diff
@@ -712,6 +712,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME:c99extidentifier)";
 				PROVISIONING_PROFILE_SPECIFIER = "";
 				SKIP_INSTALL = YES;
+				SUPPORTS_MACCATALYST = NO;
```

Comment:
I wonder what this is all about...?

Reply:
This change just disables supporting Mac Catalyst so there is one less warning. (We can't run in Mac Catalyst mode because of some of our other project settings.)